src/Editor/Editor.tsx

```diff
@@ -55,6 +63,7 @@ interface EditorProps {
   value: string;
   modifyEditor?: (monaco: typeof Monaco, editor: Monaco.editor.IStandaloneCodeEditor) => any;
   parentSize?: string;
+  editorOptions?: EditorOptions;
```

Comment:
These options are only used during the first render. Maybe it's better to call this `defaultEditorOptions` similar to how `defaultValue` is only used for first render.